eknrenderer/eknr-boxed-set-info.c

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2018 Endless Mobile, Inc. */
+
+#include "eknr-boxed-set-info.h"
+
+typedef struct _EknrBoxedSetInfoReal {
```

Comment:
Is this `Real` struct a best practice from somewhere? If not, can the refcount just go in the main struct, marked with a `/*< private >*/` comment so it's not accessible from JS?

Reply:
We could also use /*< private >*/, yes.

The `Real` struct is a pattern I've seen generally throughout GLib and Gtk - I think the reason why they use it there is both to hide things from the scanner and also to maintain ABI stability when adding private properties to structs.

I'm not sure if the latter concern really matters all that much to us, so I'm happy to just use `/*< private >*/`

Reply:
Fine either way, I agree we don't have to be too concerned about it.